rmw_connext_cpp/src/rmw_take.cpp

```diff
@@ -83,21 +103,8 @@ take(
   if (!sample_info.valid_data) {
     // skip sample without data
     ignore_sample = true;
-  } else if (ignore_local_publications) {
-    // compare the lower 12 octets of the guids from the sender and this receiver
-    // if they are equal the sample has been sent from this process and should be ignored
```

Comment:
It would be nice to retain this comment with the moved logic.

Reply:
Done in 1161405